drivers/SmartThings/zwave-switch/src/zooz-zen-30-dimmer-relay/init.lua

```diff
@@ -86,6 +83,71 @@ local function can_handle_zooz_zen_30_dimmer_relay_double_switch(opts, driver, d
   return false
 end
 
+local function find_child(parent, src_channel)
+  if src_channel == 0 then
+    return parent
+  else
+    return parent:get_child_by_parent_assigned_key(string.format("%02X", src_channel))
+  end
+end
+
+local function component_to_endpoint(device, component)
+  return { ENDPOINTS.dimmer }
+end
+
+local function endpoint_to_component(device, endpoint)
+  return "main"
+end
+
+local function do_refresh(driver, device, cmd)
+  local component = cmd and cmd.component and cmd.component or "main"
+  local endpoints = device:component_to_endpoint(component)
+  for _, ep in pairs(endpoints) do
```

Comment:
I'd use zach's suggestion to check the profile capabilities here again

Reply:
Done